source/docs/software/software-overview/control-system-software.rst

```diff
@@ -0,0 +1,133 @@
+
+LiveWindow
+----------
+
+.. image:: images/control-system-software/livewindow-smartdashboard.png
+
+LiveWindow is a mode of SmartDashboard, designed for use with the Test Mode of the Driver Station. LiveWindow allows the user to see feedback from sensors on the robot and control actuators independent of the written user code. More information about LiveWindow can be found :ref:`here <docs/software/wpilib-tools/smartdashboard/test-mode-and-live-window/index:SmartDashboard: Test Mode and Live Window>`.
+
+RobotBuilder
+------------
+
+.. image:: images/control-system-software/robot-builder.png
+
+RobotBuilder is a tool designed to aid in setup and structuring of a Command Based robot project for C++ or Java. RobotBuilder allows you to enter in the various components of your robot subsystems and operator interface and define what your commands are in a graphical tree structure.  RobotBuilder will then generate structural template code to get you started.  More information about RobotBuilder can be found :ref:`here <docs/software/wpilib-tools/robotbuilder/index:RobotBuilder>`. More information about the Command Based programming architecture can be found :ref:`here <docs/software/commandbased/index:Command-Based Programming>`.
+
+Robot Simulation
+----------------
+
+.. image:: images/control-system-software/sim-gui.png
+
+Robot Simulation offers a way for Java and C++ teams to verify their actual robot code is working in a simulated environment.  This simulation can be launched directly from VS Code and includes a 2D field that users can visualize their robot's movement on.  For more information see the :ref:`Robot Simulation section <docs/software/wpilib-tools/robot-simulation/introduction:Introduction to Robot Simulation>`.
+
+FRC LabVIEW Robot Simulator (Windows Only)
+------------------------------------------
+
+.. image:: images/control-system-software/robot-simulator.png
+
+The FRC Robot Simulator is a component of the LabVIEW programming environment that allows you to operate a predefined robot in a simulated environment to test code and/or Driver Station functions.  Information on using the FRC Robot Simulator can be found `here <https://forums.ni.com/t5/FIRST-Robotics-Competition/LabVIEW-Tutorial-10-Robot-Simulation/ta-p/3739702?profile.language=en>`__ or by opening the Robot Simulation Readme.html file in the LabVIEW Project Explorer.
+
+PathWeaver
+----------
+
+.. image:: images/control-system-software/pathweaver-5.png
+
+PathWeaver allows teams to quickly generate and configure paths for advanced autonomous routines.  These paths have smooth curves allowing the team to quickly navigate thir robot between points on the field.  For more information see the :ref:`PathWeaver section <docs/software/wpilib-tools/pathweaver/introduction:Introduction to PathWeaver>`.
+
+Robot Characterization
+----------------------
+
+.. image:: images/control-system-software/generate-project.png
+
```

Comment:
Seems like we should have at least some description here, no? Perhaps something along the lines of "This tool helps teams characterize their robot drivetrain, determining parameters that can be used for robot control."

Reply:
I got tired and skipped it when I initially added this.  I added a sentence I hope it makes sense.